### ISSUE TITLE
Issue with rm-data-access-registrar deployed using rm-data-access-core:1.2.6

### ISSUE THREAD

**Issue by dineshbkumar06** [EOEPCA/rm-data-access#20]
Initially the registrar failed to recognize http as source and this issue has been resolved with the patch https://gitlab.eox.at/vs/core/-/commit/07f7bab77a7a1ef3ae026b536abd54645c03ea3a by Fabian.

After the patch we redeployed, the rm-data-access harvester and registrar. Now we see that both modules are deployed properly with correct configuration files. 
The harvester, harvest and generates stac item for the file served by simple http server. 
The registrar now recognizes http as source and performs registration. But we notice some the following issues:

The registrar does not register the links to all the components (individual bands) of the product. Also the structure of the metadata does not look as it was with the version 1.2.5
![catalog_issue](https://user-images.githubusercontent.com/11572862/214319287-8c562a43-c375-475c-a316-e4a58f1d0ef9.png)

The http:// protocol for the files are replaced with S3://
![registrar_issue_with_storage_protocol](https://user-images.githubusercontent.com/11572862/214318712-bde107c1-8931-4f4a-83c6-4b948cf4c8b2.png)

While the output from the harvester reference the individual components properly:
![harvester_output](https://user-images.githubusercontent.com/11572862/214318921-e1d3e1fe-dde3-41a5-a49d-c9ec7b66c4cc.png)

Could you please investigate this issue?

Thanks and regards,
Dinesh Kumar Babu
adwäisEO

**Comment by rconway** [comment]
Created issue [EOEPCA-814](https://eoepca.atlassian.net/browse/EOEPCA-814).

**Comment by kalxas** [comment]
First of all it looks like you switched to the master version of the resource catalogue because I can see the stac extensions in your first screenshot.
@dineshbkumar06 can you please confirm the image versions you are using?

**Comment by kalxas** [comment]
confirmed, the 1.2.6 release is broken, it is tagged from master instead of the 1.2.x branch. I am going to delete this release now

**Comment by kalxas** [comment]
New 1.2.6 release is published just now

**Comment by dineshbkumar06** [comment]
Thank you for the quick fix. I will try the new release and let you know if everything is ok.

**Comment by kalxas** [comment]
@dineshbkumar06 please try the new 1.2.7 release for the registrar.

**Comment by dineshbkumar06** [comment]
@kalxas yes sure, I will try the new version and let you know. Thank you.

**Comment by kalxas** [comment]
Note: This issue is still going to affect the CWL registration code.

https://github.com/EOEPCA/rm-data-access/blob/master/core/registrar_pycsw/backend.py#L222
This probably needs workspace API changes to be able to fix.
